src/plugins/intel_cpu/src/ngraph_transformations/op/interaction.hpp

```diff
@@ -28,8 +28,26 @@ class InteractionNode : public ngraph::op::Op {
 
     ngraph::element::Type get_output_type() const { return m_output_type; }
 
+    void set_fq_scales(const std::vector<float>& scales) {
+        m_fq_scales = scales;
+    }
+
+    const std::vector<float>& get_output_scales() const {
+        return m_fq_scales;
+    }
+
+    void set_fq_output_type(const ngraph::element::Type& type) {
+        fq_output_type = type;
+    }
+
+    ngraph::element::Type get_fq_output_type() const {
```

Comment:
We don't need this interface. 'm_output_type' should be updated during fusing into single Interaction node.

Reply:
Sure, will move it